packages/prisma/zod-utils.ts

```diff
@@ -539,3 +539,21 @@ export const allManagedEventTypeProps: { [k in keyof Omit<Prisma.EventTypeSelect
 export const unlockedManagedEventTypeProps = {
   ...pick(allManagedEventTypeProps, ["locations", "scheduleId", "destinationCalendar"]),
 };
+
+export const bookerLayoutOptions = ["month_view", "week_view", "column_view"] as const;
+// @TODO: HAHA. Let's just look the array above and find out why that's not working..
```

Comment:
If anyone knows how to fix this? 😂
`z.union(bookerLayoutOptions.map(layout > z.literal(layout)))` didn't seem to work somehow.

Reply:
![Ill Allow It Spanish GIF](https://media4.giphy.com/media/NCjISbEPFxm48/200.gif)